build/install_deps.sh

```diff
@@ -42,6 +42,7 @@ if $LINUX; then
   $INSTALL build-essential libssl-dev libffi-dev libgmp3-dev python2.7-dev libsecret-1-dev curl
 elif $OSX && ! cmd_exists brew ; then
   /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+  brew update
```

Comment:
this only gets run if `brew` command does not exist. so it will only update when it is installed. is that what you intended?